### ISSUE TITLE
Issue Design: Screen Reader (HP,T)

### ISSUE THREAD

**Issue by jiraslama** [alan-turing-institute/AutSPACEs#325]
## Summary

This issue is to make sure that the platform can be used with a screen reader. This is to make it more accesible for a people with diverse needs, as well as to meet some of the preferences of members of the community. 

## Who can help

This requires technical skills to accomplish. As an element of the design though, anyone is welcome to review. 

## Community Sources

This was highlighted by a number of community members in focus group discussions. Some mentioned screen readers specifically, while others mentioned more generally the importance of the platform being accesible to a wide range of users with diverse needs, e.g.:

[(A d): “the words used, or any writing used” should be “well-spaced out and in a good font that anyone could read - and obviously a screen reader could pick out as well”](community-recommendations/focus-groups/20190918_summary-18-September-2019.md#user-interface)

More can be found in the [community recommendations folder](community-recommendations)

## User story

 As a: Designer  I would like to show the community a wireframe design of the screen reader functionally across components developed so far. This will demonstrate the function of the screen reader which will be activated by a screen reader toggle on / off button. The screen toggle will then be taken forward for developers to prototype as required. So that the community can provide feedback on the wireframe design, which can be taken into the prototype design stage on the project board as a concepts to ensure the platform is being developed in the correct way

Acceptance Criteria:

Review issue designs, research and wireframes take onto board for prototyping as required priority.

Define how this user story will close

Notes:

Associated pull request -  Sprint 5 MVP Structure (File Management Demo)  Components to design - wireframe and then share with developers

Current Status: Considerations Design - Design Research Wireframes Interactive & Dev Ref Research
